src/DSL.ts

```diff
@@ -0,0 +1,20 @@
+import { queryStrict } from './Query';
+import { taskEither } from 'fp-ts/lib/TaskEither';
+import { refetch } from './Strategy';
+
+export function param<A>() {
+  return queryStrict((a: A) => taskEither.of<void, A>(a), refetch);
+}
+
+export { queryShallow, queryStrict, queryJSON } from './Query';
+
+export { available, refetch, expire } from './Strategy';
```

Comment:
personally I find `available`, `refetch` e `expire` not super explicit for an API.. I would use something that vehiculate better the meaning of what they do, like: `cacheFirst`, `remoteFirst`, `timeBound` (I vaguely remember this terminology used in a Service Workers library)

Reply:
talking of which, is `refetch` actually a `remoteFirst`? Aka, if the fetch fails does it use the cache?

Reply:
> is refetch actually a remoteFirst?

no, it is a remote-only. There's no fallback encoded into the strategies at the moment, since they only affect if the value is extracted from the cache or fetched from scratch: `filter: (value: CacheValue<L, P>) => boolean`

As per the names, they are consistent with the current naming, but I'm up for changing them to something more clear eventually? Not sure if it's a good idea to this at this time, since we were originally trying to minimise the API changes

Reply:
I think that including the "meaning" of the strategies in their name would help a lot in using them wisely and better understand what are reasonable strategies to offer (for example, I believe "remote first" is a strategy that makes a lot of sense, maybe even more than "remote only"). We could start by exporting aliases to maintain backward compatibility (or not, your call)?